### ISSUE TITLE
[Phytoplankton]: 

### ISSUE THREAD

**Issue by khyde** [NOAA-EDAB/catalog#36]
### Primary Contact

kimberly.hyde@noaa.gov

### Data Name

Phytoplankton chlorophyll, size class and primary production

### Data Description

Satellite derived phytoplankton data including chlorophyll concentration, phytoplankton size class, and primary production for the Northeast Continental Shelf and ecological production units.

(To be expanded)

### Introduction to Indicator (i.e. "What is this?")

Phytoplankton support the food web as the primary food source for zooplankton and filter feeders such as shellfish. Numerous environmental and oceanographic factors interact to drive the abundance, composition, spatial distribution, and productivity of phytoplankton.

(To be expanded)

### Key Results and Visualization

(In development)

### Implications

(In development)

### Spatial Scale

By EPU and gridded for the entire shelf

### Temporal Scale

Daily, weekly, monthly, annual, climatology (1998 to current year)

### Synthesis Theme

- [X] Multiple System Drivers
- [ ] Regime Shifts
- [ ] Ecosystem Reorganization

### Define Variables

1) Chlorophyll; Concentration of chlorophyll _a_ in the near surface (first euphotic depth) waters; mg m^-3
2) Chlorophyll anomaly; Ratio of chlorophyll _a_ concentration to the long-term (1998 to present year) climatology; unitless (ratio)
3) Primary productivity; Daily net primary production of biomass expressed as carbon per unit volume in seawater per day; gCarbon m^-2 d^-1
4) Primary productivity anomaly;  Ratio of net primary production to the long-term (1998 to present year) climatology; unitless (ratio)
5) Microplankton chlorophyll; Concentration of chlorophyll _a_ in the near surface (first euphotic depth) waters from the microplankton (20-200 µm) fraction; mg m^-3
6) Nanoplankton chlorophyll; Concentration of chlorophyll _a_ in the near surface (first euphotic depth) waters from the nanoplankton (2-20 µm) fraction; mg m^-3
7) Picoplankton chlorophyll; Concentration of chlorophyll _a_ in the near surface (first euphotic depth) waters from the picoplankton (0.2-2 µm) fraction; mg m^-3
8) Microplankton fraction; The fraction of total chlorophyll concentration in the near surface (first euphotic depth) waters from microplankton (20-200 µm); (percent)
9) Nanoplankton fraction; The fraction of total chlorophyll concentration in the near surface (first euphotic depth) waters from nanoplankton (2-20 µm); (percent)
10) Picoplankton fraction; The fraction of total chlorophyll concentration in the near surface (first euphotic depth) waters from picoplankton (0.2-2 µm); (percent)
11) Annual summed primary production - TBD

### Indicator Category

- [X] Published Methods
- [ ] Extensive analysis, not yet published
- [ ] Syntheses of published information
- [ ] Database pull
- [ ] Database pull with analysis
- [X] Other

### If other, please specify indicator category

Publicly available satellite data that are processed and analyzed

### Data Contributors

Kimberly Hyde

### Point(s) of Contact

kimberly.hyde@noaa.gov

### Public Availability

Source data are publicly available.

**Comment by khyde** [comment]
It is very difficult to read the monthly time series plots.  Is it possible to reduce the white space between the plots and make the symbols smaller?  Here is an example of my original plot if that helps.
![MAB-SAV-CHLOR_A-PAN-STATS-MONTHLY_TIMESERIES](https://github.com/NOAA-EDAB/catalog/assets/5922605/0854804c-bbff-4551-8fc2-2f9327350796)

**Comment by andybeet** [comment]
We should be able to make these look a little better, yes.

**Comment by khyde** [comment]
I updated the text for the catalog page.  My recommendation for the figure layout is here https://docs.google.com/document/d/1y1oLLkCugPpQop3zrC6JK9RDNM9NS5jbJIHD3-FVvqI/edit?pli=1

**Comment by khyde** [comment]
Also, I noticed that the PSC data is incomplete for 2023 so I am updating that now.  The rest of the 2023 OC-CCI (the long-term dataset) just released so I will switch everything to that.  I will let you know when the new data have been uploaded.

**Comment by andybeet** [comment]
The figure layout wont change during this cycle. Maybe when we consolidate all of the pp/chl data into one dataset we can work something out.

**Comment by khyde** [comment]
The primary production anomaly plots should be removed.  I took another look at the data and I think there is something wrong at the anomaly calculation step and I don't have time to fix that. I would replace it with the total summed primary production (or is that already in another section?).

**Comment by jcaracappa1** [comment]
We don't have any anomaly plots in the report. For this cycle we're going to make those anomaly plots appear with the text "this indicator is currently being reviewed" instead of plotting.

**Comment by khyde** [comment]
@jcaracappa1 I was referring to the PP anomaly plots in the catalog.  Those should be removed because I believe there is an issue with the data.

**Comment by khyde** [comment]
I finished updating the CHL/PP/PSC data so that the long-term dataset (OC-CCI) is now complete through December 2023 (replacing the temporary GLOBCOLOUR data).  
https://github.com/khyde/READ-EDAB-SOE-PHYTOPLANKTON/tree/main/V2024/DATA_EXTRACTS/SOE_FORMAT

**Comment by BBeltz1** [comment]
@khyde does this include corrections to the anomaly calculations? or just an update for end of year data?

**Comment by khyde** [comment]
Just the update through the end of 2023.  There is something wrong with the annual anomaly data and that will take more time to fix and it should not be plotted.  You can plot the total annual summed primary production.

**Comment by BBeltz1** [comment]
okay, anomaly plots will be pulled from all products until your go ahead. thanks kim

**Comment by khyde** [comment]
@BBeltz1 I started making edits to this, but I'm not finished yet. I just saved it so that the content won't be lost.

**Comment by khyde** [comment]
@BBeltz1 I'm done with my edits for now, but would like to take one more look at them once the data and plots have been updated because I may need to make a couple more small edits.

**Comment by BBeltz1** [comment]
Sounds good, I will include your data update today so the page updates with the next catalog build

**Comment by khyde** [comment]
Thanks! Just let me know if you have any questions about my text updates. I did rearrange the placement and order of the plots a bit so let me know if that is an issues. Also, I was hoping that we could separate out the GB and GOM plots, but Andy said that functionality isn't an option yet. I will add that as an issue for next year.

**Comment by andybeet** [comment]
@khyde The placement of the plots is done automatically. You can not reorder. Only change text in the submission please

**Comment by khyde** [comment]
I was trying to align the order of the plots so that they match the text. I didn't realize we couldn't do text, plots and then text again in that section. I will have to rewrite the text again on Monday. I also wanted to mention that all of the GOM/GB plots except for the size class plots are shown twice.

**Comment by andybeet** [comment]
Abby picked up on the duplicate plot issue in the review. Brandon will be addressing that next week

**Comment by BBeltz1** [comment]
Yes can confirm that it's on my list of things to address in _ecodata_

**Comment by khyde** [comment]
I added 2025 and 2024 sub-headers in the Key Results and Visualizations section. I may need to do a little more text editing after I see the updated monthly time-series plots. If possible, I would really like to add a 2025 chlorophyll anomaly composite image that shows how widespread the low chlorophyll was in 2025. If we can't include it here, maybe we can add it to the 2025 highlights page (that is next on my list) and then I can just link to it. 

<img width="3419" height="1244" alt="Image" src="https://github.com/user-attachments/assets/0cc38aa4-7391-4c3f-9e73-ba7bf66ece29" />
